scripts/list_init_hooks.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/python
+#
+# Scripts for print all init hooks in f9-kernel.
+# Copyright (c) 2016 The F9 Microkernel Project. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+#
+# Authors:
```

Comment:
Don't mention "Authors" since we maintain the list in dedicated top-level files.